packages/mobx-react-lite/src/useObserver.ts

```diff
@@ -69,7 +60,8 @@ export function useObserver<T>(render: () => T, baseComponentName: string = "obs
                     // even if state did not change.
                     createReaction(adm)
                     // `onStoreChange` won't force update if subsequent `getSnapshot` returns same value.
-                    forceUpdate(Symbol())
+                    // So we make sure that is not the case
+                    adm.stateVersion = Symbol()
```

Comment:
Ehm, but you still have to call that `onStoreChange()` ... ?

Reply:
No I don't think so, I think due to uSES getting a different version from getSnapshot, it will schedule update already itself. I guess React checks that?

Reply:
(at least the test that failed without the forceUpdate, now succeeds again, the exact calling order of uSES remains a little mistery)

Reply:
Ah, yea, I think you're correct, but then the comment on line 62 is perhaps a bit misleading.

Reply:
ah yeah, didn't clean that up properly, will do in the cleanup branch